### ISSUE TITLE
Call to undefined function clone()

### ISSUE THREAD

**Issue by gwigz** [kahlan/kahlan#234]
Kahlan's been working really well for us so far, but we've hit a little struggle upon using Kahlan's autoloader (for creating more in-depth tests).

There seems to be an issue with Kahlan's autoinclude wrappers, from my guess. If `clone()` is used on an object within my code this error will be thrown, rough example:

```php
class MyClass
{
    public function foo()
    {
        $test = new MyClass; // or MyOtherClass...
        $cloned = clone($test); // error here
    }
}
```

Should (or does for me) cause this error:

```
Fatal error: Call to undefined function clone() in /private/var/folders/f_/h1gwt84n0bdfs_zy_nczrycw0000gn/T/kahlan/path/to/MyClass.php
```

May only be an issue with generated classmap autoloader stuff, doesn't seem likely though.

Also, using the `dev` branch.

**Comment by samsonasik** [comment]
can you test with space instead of parenthesis:
```php
clone $test
```
is it work with space like above?

**Comment by gwigz** [comment]
It does! 😕

**Comment by jails** [comment]
Thanks for reporting, just fixed it in the [dev](https://github.com/kahlan/kahlan/commit/834b634c76b579b3e834f5a4897cc8f508999cd6)

@gwigz, would you mind clearing all cached files updating your composer json using `"kahlan/kahlan": "dev-dev"` so I can tag a new version if ok ?

**Comment by gwigz** [comment]
Yup, that fix resolved it @jails, thanks. 👍

**Comment by jails** [comment]
Cool ! Thanks everybody, fixed in `3.1.11`
